libs/partners/elasticsearch/langchain_elasticsearch/vectorstores.py

```diff
@@ -389,6 +394,72 @@ def require_inference(self) -> bool:
         return False
 
 
+class BM25RetrievalStrategy(BaseRetrievalStrategy):
+    """Retrieval strategy using the native BM25 algorithm of Elasticsearch."""
+
+    def __init__(self, k1: float, b: float):
```

Comment:
The `k1` and `b` arguments should default to `None`.

Reply:
Thanks. I addressed in this commit: https://github.com/langchain-ai/langchain/pull/19314/commits/a2eb03a25b43994d76451c5432690971149de348